pkg/apiclient/auth_test.go

```diff
@@ -76,6 +78,7 @@ func TestApiAuth(t *testing.T) {
 	}
 
 	_, resp, err = newcli.Decisions.List(context.Background(), alert)
+	require.NoError(t, err)
```

Comment:
the CI fail because this test must return an error. As you can see below, we check that the error message contains `APIKey is empty`.